pkg/vadmin/stop_db_vc.go

```diff
@@ -55,7 +55,8 @@ func (v *VClusterOps) genStopDBOptions(s *stopdb.Parms) vops.VStopDatabaseOption
 
 	opts.DBName = v.VDB.Spec.DBName
 	opts.IsEon = v.VDB.IsEON()
-
+	opts.DrainSeconds = new(int)
```

Comment:
We should have a follow-up jira to only set drainSeconds=0 in replicated-upgrade, not for all cluster shutdown cases.